autoload/utils/exec/job.vim

```diff
@@ -51,36 +51,26 @@ function! s:createQuickFix() abort
     if l:bufnr == -1
         return
     endif
-    cexpr ''
     let l:old_error = &errorformat
     if s:err_fmt !=# ''
         let &errorformat = s:err_fmt
     endif
     execute 'cgetbuffer ' . l:bufnr
     call setqflist( [], 'a', { 'title' : s:cmake4vim_job[ 'cmd' ] } )
-    call s:closeBuffer()
     if s:err_fmt !=# ''
         let &errorformat = l:old_error
     endif
     " Remove cmake4vim job
     let s:cmake4vim_job = {}
 endfunction
 
-function! s:vimOut(channel, message) abort
-    if empty(s:cmake4vim_job) || a:channel != s:cmake4vim_job['channel']
-        return
-    endif
-    call s:appendLine(a:message)
```

Comment:
Looks like you use `s:appendLine()` anymore.

Reply:
It is used in `s:nvimOut`.

I read the documentation, `nvim` jobs cannot write to buffers, I think. I may be wrong.